### PR TITLE
[#4987]Makefile//deps/package.json//config:add jest config setup

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
need to update it as snapshot wrong

File: meinberlin/apps/livequestions/assets/CategorySelect.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import django from 'django'
 import React from 'react'
+import $ from 'jquery' // FIXME needed to run test but file should be refactored to not include jquery
```
Comment (phillimorland) [inline meinberlin/apps/livequestions/assets/CategorySelect.jsx:3]: added this as attempted several other options but without success:
- import or require via setup-jest file
- unmock and import into test file
- import or require into test file
- setting up basic jquery mock
- update funtion to not use jquery
as we want to remove jquery anyway i felt this was best option for now

Comment (khamui) [reply meinberlin/apps/livequestions/assets/CategorySelect.jsx:3]: ah i see, yes probably jQuery is too hard to mock. And refactoring to get rid of it just to write a test might be too much effort. Lets leave it for now.

File: .github/workflows/django.yml [hunk 2]
```diff
@@ -82,6 +82,14 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.COV }}
         run: |
           coveralls
+              - name: Run Frontend Tests
+        run: |
```
Comment (khamui) [inline .github/workflows/django.yml:86]: just saw that building with github actions failed. seems to be line 85 is indented too much.

**Review by khamui — COMMENTED** [review]
nice! yeah its working ⚡️

Note: except for the building on github (see comment)

**Comment by khamui** [comment]
@phillimorland look at this fancy coverage report  🤘🏼

merged! working locally and building with github actions works now.

**Comment by phillimorland** [comment]
@khamui thanks for fixing, sorry I thought I had done it?! :p

**Pull request merged**